Description: Disable udev tests
 udev is not always avaliable in Ubuntu buildds; skip tests that
 want to use this feature.
Author: Chuck Short <zulcss@ubuntu.com>
Fowarded: not-needed
--- a/neutron/tests/unit/linuxbridge/test_lb_neutron_agent.py
+++ b/neutron/tests/unit/linuxbridge/test_lb_neutron_agent.py
@@ -47,6 +47,7 @@ class FakeIpDevice(object):
 class TestLinuxBridge(base.BaseTestCase):
 
     def setUp(self):
+        self.skipTest("udev not consistently available in Ubuntu buildds")
         super(TestLinuxBridge, self).setUp()
         interface_mappings = {'physnet1': 'eth1'}
         root_helper = cfg.CONF.AGENT.root_helper
@@ -95,6 +96,7 @@ class TestLinuxBridgeAgent(base.BaseTest
         'cc:dd:ee:ff:ab:cd brd ff:ff:ff:ff:ff:ff']
 
     def setUp(self):
+        self.skipTest("udev not consistently available in Ubuntu buildds")
         super(TestLinuxBridgeAgent, self).setUp()
         # disable setting up periodic state reporting
         cfg.CONF.set_override('report_interval', 0, 'AGENT')
@@ -163,6 +165,7 @@ class TestLinuxBridgeAgent(base.BaseTest
 
 class TestLinuxBridgeManager(base.BaseTestCase):
     def setUp(self):
+        self.skipTest("udev not consistently available in Ubuntu buildds")
         super(TestLinuxBridgeManager, self).setUp()
         self.interface_mappings = {'physnet1': 'eth1'}
         self.root_helper = cfg.CONF.AGENT.root_helper
